src/tests/ui/planner/workitem-detailpage.ts

```diff
@@ -0,0 +1,13 @@
+import { ElementFinder, $ } from 'protractor';
+import { WorkItem } from './index';
+import * as ui from '../../ui';
+import { WorkItemQuickPreview } from './workitem-quickpreview';
+
+export class WorkItemDetailPage extends WorkItemQuickPreview {
+  detailPageDiv = new ui.BaseElement(this.$('#wi-detail-form'), 'wi detail page');
+
+  async isDetailPageOpened() {
```

Comment:
What is this function supposed to do?
The name `isDetailPageOpened` sounds like it would return a boolean value but it does something else. Maybe we should rename it to something more suitable.

Reply:
done